src/View.Uno.Shared/Internal/ObservableExtensions.cs

```diff
@@ -161,6 +160,8 @@ private static IObservable<bool> ObserveIsLoaded(this FrameworkElement element,
 			}
 			else
 			{
+				return Observable.FromEventPattern<THandler, TArgs>(addHandler, removeHandler, Scheduler.Immediate);
```

Comment:
Did you forget to restore the original code?

Reply:
Ohh yes! Thanks! Forgot this one.